File: eclipse/exe3.java
package eclipse;

import java.util.Scanner;

public class exe3 {
	public static void main(String[] args) {
		Scanner instance = new Scanner(System.in);
		
		System.out.println("Digite o seu nome: ");
		String name = instance.nextLine();
		System.out.println("Digite a sua idade: ");
		String age = instance.nextLine();
		System.out.println("Digite a cidade em que voce vive: ");
		String city = instance.nextLine();
		System.out.println();
		System.out.println(name+age+city);
		instance.close();
		}
}

File: logic-exercises/low-level/exe3.c
#include <stdio.h>

int main(void) {
    char name[32], city[32];
    int age;

    printf("Digite o seu nome: ");
    scanf("%s", name);
    printf("\nDigite sua idade: ");
    scanf("%d", &age);
    printf("\nDigite a cidade em que voce vive: ");
    scanf("%s", city);

    printf("%s, %d, %s", name, age, city);

    return 0;
}